book/scripts.md

```diff
@@ -659,7 +665,6 @@ first, any time we click in the page:
 ``` {.python expected=False}
 class Tab:
     def click(self, x, y):
-        while elt:
```

Comment:
Perhaps this is confusing, but I usually try to add "nesting" hints like this so readers can find the right line of code. Plus it justifies indenting things the right amount.

Reply:
I removed it because I found the "while elt" confusing when reading the chapter, as there is no indication of why it's there yet. I think it will be easy to find the code because it's in the same method.

Reply:
OK. Can we un-indent the later lines of code? (Is the differ unhappy with that?)